src/api/app/models/event/relationship.rb

```diff
@@ -1,7 +1,30 @@
 module Event
   class Relationship < Base
     self.abstract_class = true
-    payload_keys :description, :who, :user, :group, :project, :package, :role
+    payload_keys :description, :when, :who, :user, :group, :project, :package, :role, :notifiable_id
```

Comment:
Hm, `when` is always the same as `created_at` right?

Reply:
Yes, basically yes. Let me change this.

Reply:
This param equals to `created_at` now

Reply:
What I mean: Why `Relationship` or `Time.zone.now` at all? The `Event.created_at` or the `Notification.created_at` is enough...

Reply:
I think it's already using the `created_at` [here[(https://github.com/openSUSE/open-build-service/pull/12470/files#diff-1e95a883031a5d59e3bd3fcda74ecb74e3739d3cba2b164ae15fac0a63bc6970R13), right? So this `when` parameter is useless...

Reply:
yeah